editor/gui/editor_dir_dialog.cpp

```diff
@@ -204,14 +207,14 @@ EditorDirDialog::EditorDirDialog() {
 	tree = memnew(Tree);
 	vb->add_child(tree);
 	tree->set_v_size_flags(Control::SIZE_EXPAND_FILL);
-	tree->connect("item_activated", callable_mp(this, &EditorDirDialog::_item_activated));
```

Comment:
This makes it impossible to accept with double click. Maybe it could default to Move? (and could copy if you hold Ctrl, idk)

Reply:
I changed double clicking to (un)collapse that directory. Consistent with the filesystem dock.

Previously, double clicking moves/copies depending on the last action performed (because "Copy Files(s)" state is kept between runs).